overlord/state/notices.go

```diff
@@ -190,11 +190,14 @@ const (
 	// Warnings are a subset of notices where the key is a human-readable
 	// warning message.
 	WarningNotice NoticeType = "warning"
+
+	// Recorded whenever an auto-refresh is inhibited for one or more snaps.
+	RefreshInhibitNotice NoticeType = "refresh-inhibit"
 )
 
 func (t NoticeType) Valid() bool {
 	switch t {
-	case ChangeUpdateNotice, WarningNotice:
```

Comment:
I believe this will conflict with this from #13619, depending on which is merged first.

https://github.com/snapcore/snapd/blob/ebca9777aadd66d72c46bd17a6985d82e566cc9c/overlord/state/notices.go#L201-L209

Reply:
Thanks for reminding! I think I also need to account for the type checker for `refresh-inhibit`, depending on which is merged first.